modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectRelationship/SelectObjectRelationship.tsx

```diff
@@ -90,6 +90,7 @@ export default function SelectRelationship({
 		<AutoComplete<LabelNameObject>
 			emptyStateMessage={Liferay.Language.get('no-parameters-were-found')}
 			error={error}
+			id="select-object-relationship"
```

Comment:
Hi @Artartur. Please, rename the id to `objectRelationshipSelectObjectRelationship`.

Reply:
Done!